frontend/app/routes/_gcweb-app.personal-information.home-address.edit.tsx

```diff
@@ -117,26 +117,16 @@ export default function PersonalInformationHomeAddressEdit() {
     };
   }) as InputOptionProps[];
 
-  const regions: InputOptionProps[] = selectedCountry
-    ? (countryRegions
-        .map((region) => {
-          return {
-            label: i18n.language === 'fr' ? region.nameFr : region.nameEn,
-            value: region.code,
-            id: region.code,
-          };
-        })
-        .sort((r1, r2) => (r1.id < r2.id ? -1 : 1)) as InputOptionProps[])
-    : (regionList
-        .filter((region) => region.country.code === defaultValues.country)
-        .map((region) => {
-          return {
-            label: i18n.language === 'fr' ? region.nameFr : region.nameEn,
-            value: region.code,
-            id: region.code,
-          };
-        })
-        .sort((r1, r2) => (r1.id < r2.id ? -1 : 1)) as InputOptionProps[]);
+  //populate region/province/state list with selected country or current address country
+  const regions: InputOptionProps[] = (selectedCountry ? countryRegions : regionList.filter((region) => region.country.code === defaultValues.country))
+    .map((region) => {
+      return {
+        label: i18n.language === 'fr' ? region.nameFr : region.nameEn,
+        value: region.code,
+        id: region.code,
+      };
+    })
+    .sort((r1, r2) => (r1.id < r2.id ? -1 : 1)) as InputOptionProps[];
```

Comment:
**Suggestion:** For all select options, we should be sorting by `label` and taking into account the accented characters of the `label` while sorting (ie. using [`localeCompare`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare)).

Reply:
updated